class.jetpack-gutenberg.php

```diff
@@ -23,7 +23,9 @@ function jetpack_register_block( $slug, $args = array() ) {
 		_doing_it_wrong( 'jetpack_register_block', 'Prefix the block with jetpack/ ', '7.1.0' );
 		$slug = 'jetpack/' . $slug;
 	}
-	return register_block_type( $slug, $args );
+
+	// Checking whether block is registered to ensure it isn't registered twice.
+	return Jetpack_Gutenberg::is_registered( $slug ) ? false : register_block_type( $slug, $args );
```

Comment:
Any specific reason to do it all on one line in this case? I don't mind either way, but I think we tend to usually be more verbose / explicit and split those things onto 2 lines. what do you think?